### ISSUE TITLE
Discuss type of statistics to display on landing page

### ISSUE THREAD

**Issue by timeu** [1001genomes/AraGWAS#24]
Currently we only display the top genes, but we could display much more information also over time

**Comment by mtog** [comment]
Yes, I think this will become clearer once we have all the associations loaded online.
